geniza/corpus/views.py

```diff
@@ -746,7 +746,11 @@ def get_context_data(self, **kwargs):
                     "csrf_token": csrf_token(self.request),
                 },
                 "tiny_api_key": getattr(settings, "TINY_API_KEY", ""),
-                "source_detail": f"{source} {source.notes}." if source else "",
+                "source_detail": mark_safe(
+                    f"{source.formatted_display()} {source.notes}."
```

Comment:
Ah, ok! No, we're not loading the footnote in this view at the moment. I'll remove the notes and just use the source formatted display.

Reply:
for context, the reason I think we should display it here: that's where we have notes like "emendations by Alan Elbaum" when Alan has made improvements or corrections to an existing transcription by Goitein, so it's transcription/footnote specific, not source

Reply:
That makes sense. I can add a TODO here. The digital edition might not actually be created until after the first annotation is made, right? I guess we would just `pass` if the digital edition footnote does not exist, and otherwise add the notes to the context. Should I go ahead and do that or would you prefer a TODO and handling it later?

Reply:
ah good point; yes, it might not exist yet for a new transcription

TODO is good, let's address on the next round of work after this round of acceptance testing.